interfaces/python/CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ else()
 
             set(CMAKE_SWIG_FLAGS "-py3")
 
+            if(SWIG_VERSION VERSION_GREATER_EQUAL "4.0.0")
```

Comment:
I don't think this comparison is available in cmake 3.4 so I don't think we can use it until HELICS 3.0   may just need to use SWIG_VERSION VERSION_GREATER 3.9 or something like that so 4.0 works but any of the 3.X series don't, since I don't believe swig ever had a 3.2 or higher

Reply:
Ah that's annoying. Okay, I've changed it back to `VERSION_GREATER` but left it at 4.0.0.

Resolved in 7639096280c260a3e90720497aa94e08f09440ce.